docs/source-pytorch/starter/installation.rst

```diff
@@ -6,6 +6,14 @@
 Installation
 ############
 
+*************
```

Comment:
Could we avoid this section, but integrate the sentence below into the "Custom PyTorch Version" section?

Reply:
Hello @awaelchli,
Sure! I've updated the PR.
Thanks for your feedback!